### ISSUE TITLE
Field with `accessor` keyword in class doesn't report an error in ts playground but will in ast viewer

### ISSUE THREAD

**Issue by yuhuung** [microsoft/TypeScript#55536]
### 🔎 Search Terms

`accessor`

### 🕗 Version & Regression Information

The code snippet won't report any error in typescript playground (eg. v4.9), but will report error when in https://ts-ast-viewer.com/

### ⏯ Playground Link

https://www.typescriptlang.org/play?strict=false&noImplicitAny=false&strictNullChecks=false&strictFunctionTypes=false&strictPropertyInitialization=false&strictBindCallApply=false&noImplicitThis=false&ts=4.9.5#code/MYGwhgzhAEAKCmAnCB7AdtA3gKGn6Yww8UKi0aYAtvAFwQAuiAlmgObYC+QA

### 💻 Code

```ts
class Person {
    accessor name:string
}
```

### 🙁 Actual behavior

The behavior in typescript playground and https://ts-ast-viewer.com/ is not the same, which cause I can't tell if the code is correct.
- Typescript playground
![image](https://github.com/microsoft/TypeScript/assets/43562440/829c2f4f-2330-471b-a55e-b05ae7fe5761)
- Ast Viewer
![image](https://github.com/microsoft/TypeScript/assets/43562440/dc18ea47-b4b2-4576-bb92-b5d7df33014d)

### 🙂 Expected behavior

The behavior should be the same so that I can tell which one is right.

**Comment by MartinJohns** [comment]
Seems to be an issue with the TypeScript AST Viewer, so perhaps create an issue in their repository? Doesn't seem to be a problem with TypeScript.

**Comment by yuhuung** [comment]
Got it, thanks!
